templates/project.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# SPDX-FileCopyrightText: 2023 CERN (home.cern)
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
```

Comment:
This file should also have a version field in my opinion.

Reply:
This template will not be distributed, it will only be used in this repository. If we change the syntax, we can simply update the existing configuration files instead of keeping backward compatibility in the generator. I removed the version because of that, do you think I should add it back?

Reply:
I agree with you, no version here